### PR TITLE
Update ares-log help message

### PR DESCRIPTION
**Pull request by ssuminahn**
:Release Notes:
Update help of ares-log

:Detailed Notes:
Update help of ares-log

:Testing Performed:
1. Pass unit test on ose and auto target
2. Pass eslint
3. Check ares-log help message
 - ares-log -h

:Issues Addressed:
[PLAT-143924] Update ares-log help message

### CODE REVIEW

**Review by ssuminahn — COMMENTED** [review]
code :+1: 
verify :+1:

File: files/help/ares-log.help [hunk 1]
```diff
@@ -24,67 +24,62 @@
...
-        "ares-log -pid 1735 -s pid_1735.log",
+        "# Show logs of specified process ID and save them to a log file",
+        "ares-log -pid 1735 -s pid_1735.log -d DEVICE",
         "",
-        "# Show unit list of host",
-        "ares-log -ul",
-        "",
-        "# Show logs from sam of display 1",
-        "ares-log -u sam -dp 1",
+        "# Show logs of SAM",
```
Comment (YoungeunKim) [inline files/help/ares-log.help:59]: @ssuminahn , Show logs of UNIT??

**Review by YoungeunKim — CHANGES_REQUESTED** [review]
Code :-1:

**Review by ssuminahn — COMMENTED** [review]
code :+1: 
verify :+1:

File: files/help/ares-log.help (hunk 1 above)
Comment (ssuminahn) [reply files/help/ares-log.help]: I've updated this.

**Review by YoungeunKim — APPROVED** [review]
Code :+1: 
Verified :+1:

**Comment by YoungeunKim** [comment]
I will merge this commit after developer branch is ready to release CLI 2.2.0.

**Comment by ssuminahn** [comment]
> I will merge this commit after developer branch is ready to release CLI 2.2.0.

@YoungeunKim 
I think you left your comment wrong. Please check this.

**Comment by YoungeunKim** [comment]
> > I will merge this commit after developer branch is ready to release CLI 2.2.0.
> 
> @YoungeunKim
> I think you left your comment wrong. Please check this.

Removed original comment

**Pull request merged**